model/load_data.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+from collections import defaultdict
+from datetime import datetime, timedelta
+from copy import deepcopy
+import os
+from model.constants import *
+
+
+class CLData:
+    def __init__(self):
+        self.pr_level_features = None
+        self.file_level_features = {}
+        self.reverted = False
+
+
+class DataLoader:
+    def __init__(self, repos):
+        self.repos = repos
+        self.pr_columns = PR_LEVEL_FEATURES + ['num new files', 'num files',
```

Comment:
Why you can't add these extra features to PR_LEVEL_FEATURES constant?

Reply:
Fixed.